docs/source/installation/login.md

```diff
@@ -37,17 +37,32 @@ Login using the username `root` and the password provided for the initial Keyclo
 
 All QHub users will be part of the `qhub` realm (a realm is a distinct identity provider and set of users in Keycloak). Note that the root user alone is a member of the `master` realm.
 
-The qhub realm should be selected by default.
+The `qhub` realm should be selected by default.
 
 Click 'Users' along the left-hand side of the page.
 
 Click the 'Add user' button and you will see the new user form:
 
-![Add User to Keycloak](../images/keycloak_adduser.png)
+![Add User to Keycloak](../images/keycloak_add_users.png)
 
-Enter the name you would like for the user then click Save.
+There are three fields, outline above, which should be filled out.These are 'Username', 'Email', and 'Groups'.
 
-Once the user has been created, you can set a password (not needed for GitHub/Auth0 login):
+Depending on the authentication provider selected ('password', 'GitHub' or 'Auth0'), the values entered into the 'Username' field will differ slightly. The following table outlines those differences:
```

Comment:
```suggestion
Depending on the authentication provider selected ('password', 'GitHub' or 'Auth0'), the values entered into the 'Username' field will differ slightly based on the case. The following table outlines those differences:
```

Reply:
Personally this wording sounds a bit off. I'll see if I can't rework it.